### PR TITLE
Add scenario 'check resume info in rhel6' for v2v case134430

### PR DESCRIPTION
**Pull request by mxie91**
Signed-off-by: mxie91 <mxie@redhat.com>

### CODE REVIEW

**Comment by mxie91** [comment]
# avocado run --vt-type v2v function_test_esx.positive_test.resume_rhel7.esx_65.rhev.rhv_upload.it_vddk.vpx_uri
JOB ID     : 8b9bda77e8c905886e0bb7f36b8ebac290f8ed27
JOB LOG    : /root/avocado/job-results/job-2020-04-15T10.52-8b9bda7/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.resume_rhel7.esx_65.rhev.rhv_upload.it_vddk.vpx_uri: PASS (796.57 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 797.57 s

# avocado run --vt-type v2v function_test_esx.positive_test.resume_rhel6.esx_67.rhev.rhv_upload.it_vddk.vpx_uri
JOB ID     : 0b6df4868df52cd4b3b8c4a9e80da9bbec5139c6
JOB LOG    : /root/avocado/job-results/job-2020-04-16T11.31-0b6df48/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.resume_rhel6.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (526.39 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 527.37 s

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -147,11 +148,10 @@ def check_resume_swap(vmcheck):
         """
         Check the content of grub files meet expectation.
         """
-        # Only for grub2
-        chkfiles = ['/etc/default/grub',
-                    '/boot/grub2/grub.cfg',
-                    '/etc/grub2.cfg']
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:153]: Doesn't the boot grub configuration need to be checked?
For grub2, /etc/grub2.cfg
For grub, /etc/grub.cfg

Comment (mxie91) [reply v2v/tests/src/function_test_esx.py:153]: I will add them

**Comment by xiaodwan** [comment]
@chloerh Please help to review this PR, thanks.

**Pull request merged**